- Do not install package-lock.json into the wheel
  * boo#1231254 gh#openSUSE/obs-service-node_modules#41
- Require typescript for compilation
- Fix test artifact destination

Index: panel-1.7.1/pyproject.toml
===================================================================
--- panel-1.7.1.orig/pyproject.toml
+++ panel-1.7.1/pyproject.toml
@@ -111,7 +111,7 @@ raw-options = { version_scheme = "no-gue
 
 [tool.hatch.build.targets.wheel]
 include = ["panel"]
-exclude = ["panel/node_modules"]
+exclude = ["panel/node_modules", "panel/package-lock.json"]
 
 [tool.hatch.build.targets.wheel.force-include]
 "panel/dist" = "panel/dist"
Index: panel-1.7.1/panel/tests/command/test_compile.py
===================================================================
--- panel-1.7.1.orig/panel/tests/command/test_compile.py
+++ panel-1.7.1/panel/tests/command/test_compile.py
@@ -21,7 +21,7 @@ class JSTestComponentWithShared(JSCompon
 
 def test_compile_component(py_file):
     cmd = [sys.executable, "-m", "panel", "compile", "panel.tests.command.test_compile:JSTestComponent", "--unminified"]
-    p = subprocess.Popen(cmd, shell=False, cwd=CWD)
+    p = subprocess.Popen(cmd, shell=False)
     p.wait()
 
     bundle = CWD / "JSTestComponent.bundle.js"
@@ -35,7 +35,7 @@ def test_compile_component(py_file):
 
 def test_compile_component_with_shared_file(py_file):
     cmd = [sys.executable, "-m", "panel", "compile", "panel.tests.command.test_compile:JSTestComponentWithShared", "--unminified"]
-    p = subprocess.Popen(cmd, shell=False, cwd=CWD)
+    p = subprocess.Popen(cmd, shell=False)
     p.wait()
 
     bundle = CWD / "JSTestComponentWithShared.bundle.js"
